docs/solutions-ecommerce.mdx

```diff
@@ -7,7 +7,26 @@ date: 2022-04-13
 tags: ["ecommerce solution", "ecommerce search ui", "overview"]
 ---
 
-- talk about how to build great ecommerce experiences relies on having:
-  - great autocomplete
-  - Product Listing Pages (e.g. /category/clothing)
-  - Recommending products within product detail pages (e.g. /product/clothing/polo-shirt)
+In ecommerce, to build great search experiences for your customers, you need to focus on three major experiences:
```

Comment:
This sentence mentions three parts, but below we describe four, and we have five pages in the navigation. Do you think we should have five items everywhere?

I also think the image is a bit out of place here — it interrupts the text in the middle and brings unrelated concepts, like highlighting. Maybe we could put it in a separate section?